lib/CryptoNoteCore/Currency.cpp

```diff
@@ -166,8 +167,18 @@ bool Currency::getBlockReward(
         penalizedFee = getPenalizedAmount(fee, medianSize, currentBlockSize);
     }
 
+    uint64_t consistency = 1;
```

Comment:
Change this to double, because with integer you will get either 0, 1 or 2.

Reply:
Thanks for noticing this tiny bug! 🙂

Reply:
Fixed in the latest commit in this PR.